tests/unit_tests/test_recursive_dbscan.py

```diff
@@ -104,6 +116,20 @@ def test_binning_invalid_clustering_method(master, markers):
         )
 
 
+@pytest.mark.wip
+def test_binning_empty_markers_table(
+    df_without_markers, master,
+):
+    with pytest.raises(TableFormatError):
+        recursive_dbscan.binning(
+            master=master,
+            markers=df_without_markers,
+            domain="bacteria",
+            method="hdbscan",
+            taxonomy=False,
+        )
```

Comment:
We may be over using fixture here. Since this `df_without_markers` fixture is only used once, we can just define it within this test function. For one, this enhances readability and two, we can try to prevent any unintended side effects that may be caused by any particular fixture.